static/js/directives/ui/repo-tag-history.js

```diff
@@ -150,6 +163,11 @@ angular.module('quay').directive('repoTagHistory', function () {
         }
       };
 
+      $scope.isFuture = function(entry) {
+        if (!entry) { return false; }
+        return entry.time >= new Date().getTime();
+      };
+
       $scope.getEntryClasses = function(entry, historyFilter) {
```

Comment:
You pass 3 arguments in the AngularJS template, but there are only 2 parameters here.